app/views/posts/show.html.slim

```diff
@@ -4,29 +4,32 @@
 = render partial: "awards_modal"
 
 .dflex.justify-content-center
+  - if @post.banner.present?
+    = image_tag @post.banner, class: "card-img-top"
   .card.post
     small.text-muted
-      | Posted by
+      | Posted by&#160
       i
         = @post.user.username
     h4.card-title
       = @post.title
       - if user_signed_in?
          i#bookmark_icon.bi class="#{@bookmarked ? 'bi-bookmark' : 'bi-bookmark-fill'}" data-id="#{@post.id}"
          i#award_icon.bi.bi-gift data-bs-toggle="modal" data-bs-target="#awardsSelectModal"
+    .d-flex.flex-row-reverse
+      - if policy(@post).edit?
+        = link_to edit_community_post_path(@post.community.id, @post.id), method: :get, class: "action-button" do
+          i.bi.bi-pencil
+      - if policy(@post).destroy?
+        = link_to community_post_path(@post.community.id, @post.id), data: { confirm: "Are you sure?" }, method: :delete, class: "action-button" do
```

Comment:
izmēģini šādi: `link_to [@post.community, post]......`
starp citu ja tev nav nepieciešams community, tad to var arī neielādēt tiekai priekš linka vajadzībām:
```
@post.community.id -> trigerē community queriju
@post.community_id -> netrigerē community queriju
```